README.md

```diff
@@ -1,30 +1,34 @@
-# nwoc_algorithms
-This is a library of various algorithms that are used in competitive programming.
```

Comment:
why remove?

Reply:
With everyone making pull requests and updating the readme my pull request was showing some conflicts.So I replaced some data in readme and added link to the new Egyptian Fraction Algorithm in the readme. I did this to resolve the conflicts.